src/serverEntry.tsx

```diff
@@ -60,6 +68,14 @@ const server = createKoaServer({
 })
 
 server.router.get('/', getPage)
+reactPageRoutes.forEach((route) => {
+  server.router.get(route.path, getPage)
+})
+
+server.app.use(proxy({
+  target: 'http://localhost:8443',
```

Comment:
I think this value should be configurable via environment variable.

```suggestion
  target: process.env.API_URL
```